website/project/model.py

```diff
@@ -1558,10 +1567,10 @@ def set_title(self, title, auth, save=False):
         :param auth: All the auth information including user, API key.
         """
         #Called so validation does not have to wait until save.
-        validate_title(title)
+        new_title = validate_title(title)
```

Comment:
It might help to separate out what's going on.

"Validation" happens at save. It should either allow something to be saved, or not.

"Sanitization" happens before saving. It is what alters the text going into the database.

Are we calling validation here for any reason other than to make sanitization occur? (the commit logs may be helpful in determining when the comment line was added)